config.js

```diff
@@ -42,6 +40,7 @@ System.config({
     "babel": "npm:babel-core@5.8.21",
     "babel-runtime": "npm:babel-runtime@5.8.20",
     "bootstrap": "github:twbs/bootstrap@3.3.5",
+    "clean-css": "npm:clean-css@3.3.8",
```

Comment:
We need `clean-css` package for [bootstrap.css](https://github.com/aurelia/skeleton-navigation/blob/master/src/app.js#L2) bundling to work properly.